### PR TITLE
Create Nofap.com.xml

### PR DESCRIPTION
**Pull request by HTTPSNowhere**

### CODE REVIEW

File: src/chrome/content/rules/Nofap.com.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	Nonfunctional domains:
+
+		- war	(refused)
+
+-->
+<ruleset name="Nofap.com">
+	<target host="nofap.com" />
+	<target host="www.nofap.com" />
+	<target host="store.nofap.com" />
+
```
Comment (koops76) [inline src/chrome/content/rules/Nofap.com.xml]: Add `<securecookie host=".+" name=".+" />`.

Comment (HTTPSNowhere) [inline src/chrome/content/rules/Nofap.com.xml]: @koops76 Done!

**Comment by HTTPSNowhere** [comment]
Partially fixes: https://github.com/EFForg/https-everywhere/issues/5055

**Comment by jeremyn** [comment]
@HTTPSNowhere Are you still interested in working on this pull request?

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @HTTPSNowhere . I've noted `nofap.com` at `https://github.com/EFForg/https-everywhere/issues/3069#issuecomment-353648503` in case someone else wants to make a pull request for this domain.

**Comment by J0WI** [comment]
> @HTTPSNowhere Are you still interested in working on this pull request?

@jeremyn I think you closed that one to early. There was no pending feedback to continue working on.

**Comment by jeremyn** [comment]
@J0WI It is a waste of time to provide feedback on a PR if the contributor has abandoned it.

If you want to talk about how this works in more detail, let's do that in issue https://github.com/EFForg/https-everywhere/issues/13859.

**Pull request closed without merging**